plugins/characters/plugin/views/cl_character_general.lua

```diff
@@ -51,7 +52,7 @@ function PANEL:Init()
     end
   end
 
-  self.gender_female = vgui.Create('fl_button', self)
+  self.gender_female = vgui.Create('fl_button', self) 
```

Comment:
Can you please remove this whitespace at the end of line?

Reply:
done! commit 4923d12eca78b1e7944754fec462c53a77e627b1